example/example.go

```diff
@@ -19,23 +19,23 @@ func main() {
 
 	// In production no tokens should be logged
 	logData := log.Data{"address": devAddress, "token": token}
-	log.Event(nil, "Created vault client", log.INFO, logData)
+	log.Info(nil, "Created vault client", logData)
```

Comment:
I'd take this opportunity to change the `nil` context to `context.Background`. I think we're not meant to pass `nil` context https://pkg.go.dev/context 

I realise this is just an example, so I won't block the PR!

Reply:
Yeah, it's better to show correct usage in an example anyway. Fixed.